Lib/glyphsets/test_strings.json

```diff
@@ -1,35 +1,35 @@
 {
-    "GF_Latin_Kernel": {
+    "GF_Latin_Core": {
         "Letter Groups": [
-            "EFHIJLTU", 
-            "AVWXZ",
-            "KMNY",
-            "COQS",
-            "BPRDG",
-            "fijltr",
-            "kvwxyz",
-            "hmnu",
-            "ceogs",
-            "abdpq"
+            "EFHIJLTU AVWXZ KMNY",
+            "COQS BPRDGÞẞ",
+            "hmnu ceogðsß abdpqþ",
+            "kvwxyz fijltr",
+            "0123456789",
+            ".,:;/|?!-–—&@%([{}])°",
+            "+−×÷=≠><≥≤±≈~¬∞"
         ],
         "Spacing": [
             "HHOHHOOHOO HIHOIO",
             "uuonouonoo ninoioolonlniuo",
-            "010 080 030 020 050 060 070"
+            "010 080 030 020 050 060 070",
+            "3+4=7 5>2 2x²−(8×7)÷9≈1",
+            "ĿL·L ŀl·l"
         ],
         "Kerning": [
-            "HHAVAHH WAFAPATAUAYALYTJĿLVÆ HH",
-            "TiTíTîTïTìTĭTīTĩT Vă Tä Tü Tõ Tŭ Tř",
-            "HHTan oTo yTy vAv oVo eYe Ly Ky Ac",
+            "HHAVAHH OXOWAFAPATAUAYAÞYLYTJLVÆAVAOYO HH",
+            "Ti Tí Tî Tï Tì Tī Vă Tä Tü Tõ Tŭ Tř",
+            "HHTan oTo yTy vAv oVo oWo eYe Ly Ky Ac Ay gj",
             "nin non nvn ovo oxo ayn fl fi fj ft gj ko rento",
-            "ďá ďu ŀl gj ľk włw yły ółr fħ ílïlîl",
+            "ďá ďu gj ľk włw yły ółr fħ ílïlîl",
             "H.Y-Y.T,F.P,V:Y„V-T-A*A’A“A’A-AL–L—L-",
             "n.r.r,y.y,(o)(i)(d)(j)(f)[j]{f}",
-            "080 076 474 973 94 7078",
+            "¿o,O? w@y «n•n» n*",
+            "080 076 474 973 94 7078 54 24 272 679",
             "P4T47A .47,9.7-»4",
             "42°21′29″N 71°03′49″W",
```

Comment:
Minute and second (2032 and 2033) ar not actually included in the GF Latin Core. We should either add them there or remove them from here. I would advocate for the before (adding them into the glyphset) wdyt?

Reply:
I don't know what is best. I would see them in Core, but they can be replaced by ' and " so I don't know how essential they are. Maybe we can add an entry GF_Latin_Plus in there and add test for the different symbols (like the bullets are the same width etc) so it may motivates designer to add all these symbols when they see notdef?

Reply:
I've been motivating designers to add minute and second following the reported notdef. But I've been saying "despite they are not required in Latin Core, this would be a good addition etc" Precisely cause they differ from quotation marks and could be really used. 

So at least for these two we could add them to Core, or change this string to use the quotes instead, so notdef reported.
And eventually, yes, also add a .json file for Latin Plus which is more related to symbols and others, like Vietnamese